frontend/content/resources/vscode-cdk-snippets/index.yml

```diff
@@ -0,0 +1,8 @@
+title: CDK Construct Snippets for VS Code
+teaser: This extension adds L1 construct snippets from AWS CDK into Visual Studio Code.
+url: "https://marketplace.visualstudio.com/items?itemName=dannysteenman.cdk-snippets"
+createdAt: "2021-05-04 20:00"
```

Comment:
I'd recommend making this the current date so that you're bumped up to the top of the list =-}

Reply:
ah good one, didn't know that! Let me update it :)